### PR TITLE
HNT max supply (HIP draft)

### PR DESCRIPTION
**Pull request by jmfayal**
Rendered: https://github.com/jmfayal/HIP/blob/patch-2/0020-hnt-max-supply.md

### CODE REVIEW

**Comment by jamiew** [comment]
Proposal looks good. Per discussion in Discord and Helium Inc RFC gonna go ahead and merge for official discussion 👍

**Comment by jamiew** [comment]
19 is technically open but we've already called this 20 so I'm just going to slot the next draft-merge in at 19. Hopefully the HIP gods forgive me

File: 0020-hnt-max-supply.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+All HNT holders, Hotspot owners, and HST holders will be affected by this HIP. Users of the network will see no change to data transmission pricing.
+
+# Detailed Explanation
+
+The proposal is to have halvenings of net HNT issuance every 2 years on the anniversary of genesis. This means the first halvening will be on 8/1/2021 and net HNT issuance will be reduced to 2.5M HNT per month.
+
+The HNT mining split amongst stakeholders will not change from the [current schedule](https://dl.dropboxusercontent.com/s%2Ffyn62o2ea8k1l41%2FScreen%2520Shot%25202020-10-31%2520at%252009-46-50%2520The%2520New%2520Wireless%2520Economy.%2520.png):
+
+![|672x556](https://lh3.googleusercontent.com/wTISjqJrP_kLzAjgVN8YqWEAszUJ3BoMGBpdMzevXa92FL0HfmQwlS9E4hZXCFhHw8iv2KTcfpc5cy-MdwOEREsWNN4Uwtp13dMXNHI-qpXTSMcka6EgGSrLqhQlX2wY_FU2NwA)
```
Comment (abhay) [inline 0020-hnt-max-supply.md:32]:
```suggestion
|-|HNT at start of year | Total HNT Minted | % to Proof of Coverage (+ any extra from Data Transfer) | % to Data Transfer (excess to Proof of Coverage) | % to Founders Reward | % to Consensus |  | # of hotspots on network | HNT Price | Hotspot cost | 
|-|-|-|-|-|-|-|-|-|-|-|
 |Year 1 | 0 | 5,000,000.0 | 30.00% | 30.00% | 35.00% | 5.00% |  | 5,000 | $0.25 | $500 | 
 |Year 2 | 5,000,000 | 5,000,000.0 | 28.50% | 32.50% | 34.00% | 5.00% |  | 10,000 | $1.00 | $250 | 
 |Year 3 | 10,000,000 | 2,500,000.0 | 27.00% | 35.00% | 33.00% | 5.00% |  | 50,000 | $1.00 | $150 | 
 |Year 4 | 12,500,000 | 2,500,000.0 | 25.50% | 37.50% | 32.00% | 5.00% |  | 100,000 | $1.00 | $100 | 
 |Year 5 | 15,000,000 | 1,250,000.0 | 24.00% | 40.00% | 31.00% | 5.00% |  | 150,000 | $1.00 | $100 | 
 |Year 6 | 16,250,000 | 1,250,000.0 | 22.50% | 42.50% | 30.00% | 5.00% |  | 200,000 | $1.00 | $100 | 
 |Year 7 | 17,500,000 | 625,000.0 | 21.00% | 45.00% | 29.00% | 5.00% |  | 250,000 | $1.00 | $100 | 
 |Year 8 | 18,125,000 | 625,000.0 | 19.50% | 47.50% | 28.00% | 5.00% |  | 300,000 | $1.00 | $100 | 
 |Year 9 | 18,750,000 | 312,500.0 | 18.00% | 50.00% | 27.00% | 5.00% |  | 350,000 | $1.00 | $100 | 
 |Year 10 | 19,062,500 | 312,500.0 | 16.50% | 52.50% | 26.00% | 5.00% |  | 400,000 | $1.00 | $100 | 
 |Year 11 | 19,375,000 | 156,250.0 | 15.00% | 55.00% | 25.00% | 5.00% |  | 450,000 | $1.00 | $100 | 
 |Year 12 | 19,531,250 | 156,250.0 | 13.50% | 57.50% | 24.00% | 5.00% |  | 500,000 | $1.00 | $100 | 
 |Year 13 | 19,687,500 | 78,125.0 | 12.00% | 60.00% | 23.00% | 5.00% |  | 550,000 | $1.00 | $100 | 
 |Year 14 | 19,765,625 | 78,125.0 | 10.50% | 62.50% | 22.00% | 5.00% |  | 600,000 | $1.00 | $100 | 
 |Year 15 | 19,843,750 | 39,062.5 | 9.00% | 65.00% | 21.00% | 5.00% |  | 650,000 | $1.00 | $100 | 
 |Year 16 | 19,882,813 | 39,062.5 | 7.50% | 67.50% | 20.00% | 5.00% |  | 700,000 | $1.00 | $100 | 
 |Year 17 | 19,921,875 | 19,531.3 | 6.00% | 70.00% | 19.00% | 5.00% |  | 750,000 | $1.00 | $100 | 
 |Year 18 | 19,941,406 | 19,531.3 | 4.50% | 72.50% | 18.00% | 5.00% |  | 800,000 | $1.00 | $100 | 
 |Year 19 | 19,960,938 | 9,765.6 | 3.00% | 75.00% | 17.00% | 5.00% |  | 850,000 | $1.00 | $100 | 
 |Year 20 | 19,970,703 | 9,765.6 | 1.50% | 77.50% | 16.00% | 5.00% |  | 900,000 | $1.00 | $100 | 
 |Year 21 | 19,980,469 | 4,882.8 | 0.00% | 80.00% | 15.00% | 5.00% |  | 950,000 | $1.00 | $100 | 
 |Year 22 | 19,985,352 | 4,882.8 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,000,000 | $1.00 | $100 | 
 |Year 23 | 19,990,234 | 2,441.4 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,050,000 | $1.00 | $100 | 
 |Year 24 | 19,992,676 | 2,441.4 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,100,000 | $1.00 | $100 | 
 |Year 25 | 19,995,117 | 1,220.7 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,150,000 | $1.00 | $100 | 
 |Year 26 | 19,996,338 | 1,220.7 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,200,000 | $1.00 | $100 | 
 |Year 27 | 19,997,559 | 610.4 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,250,000 | $1.00 | $100 | 
 |Year 28 | 19,998,169 | 610.4 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,300,000 | $1.00 | $100 | 
 |Year 29 | 19,998,779 | 305.2 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,350,000 | $1.00 | $100 | 
 |Year 30 | 19,999,084 | 305.2 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,400,000 | $1.00 | $100 | 
 |Year 31 | 19,999,390 | 152.6 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,450,000 | $1.00 | $100 | 
 |Year 32 | 19,999,542 | 152.6 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,500,000 | $1.00 | $100 | 
 |Year 33 | 19,999,695 | 76.3 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,550,000 | $1.00 | $100 | 
 |Year 34 | 19,999,771 | 76.3 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,600,000 | $1.00 | $100 | 
 |Year 35 | 19,999,847 | 38.1 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,650,000 | $1.00 | $100 | 
 |Year 36 | 19,999,886 | 38.1 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,700,000 | $1.00 | $100 | 
 |Year 37 | 19,999,924 | 19.1 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,750,000 | $1.00 | $100 | 
 |Year 38 | 19,999,943 | 19.1 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,800,000 | $1.00 | $100 | 
 |Year 39 | 19,999,962 | 9.5 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,850,000 | $1.00 | $100 | 
 |Year 40 | 19,999,971 | 9.5 | 0.00% | 80.00% | 15.00% | 5.00% |  | 1,900,000 | $1.00 | $100 | 
 |Year 41 | 19,999,981 | 4.8 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 42 | 19,999,986 | 4.8 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 43 | 19,999,990 | 2.4 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 44 | 19,999,993 | 2.4 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 45 | 19,999,995 | 1.2 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 46 | 19,999,996 | 1.2 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 47 | 19,999,998 | 0.6 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 48 | 19,999,998 | 0.6 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 49 | 19,999,999 | 0.3 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 50 | 19,999,999 | 0.3 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 51 | 19,999,999 | 0.1 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 52 | 20,000,000 | 0.1 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 53 | 20,000,000 | 0.1 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 54 | 20,000,000 | 0.1 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 55 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 56 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 57 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 58 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 59 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 60 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 61 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 62 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 63 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 64 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 65 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 66 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 67 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 68 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 69 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 70 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 71 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 72 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  | 
 |Year 73 | 20,000,000 | 0.0 | 0.00% | 80.00% | 15.00% | 5.00% |  |  |  |  |
```

**Comment by jamiew** [comment]
Shoutout to @abhay for the table-table 💯

**Comment by jamiew** [comment]
This HIP draft has been reviewed, numbered, and merged for discussion. Please direct feedback & questions to its tracking issue: https://github.com/helium/HIP/issues/73

**Pull request merged**